src/Intl/Idn/Idn.php

```diff
@@ -147,7 +147,7 @@ final class Idn
      */
     public static function idn_to_ascii($domainName, $options = self::IDNA_DEFAULT, $variant = self::INTL_IDNA_VARIANT_UTS46, &$idna_info = [])
     {
-        if (\PHP_VERSION_ID >= 70200 && self::INTL_IDNA_VARIANT_2003 === $variant) {
+        if (\PHP_VERSION_ID >= 70400 && self::INTL_IDNA_VARIANT_2003 === $variant) {
```

Comment:
But PHP itself triggers the deprecation on PHP 7.3: https://3v4l.org/ghvgU
So I'm not sure this is correct.